Lib/subprocess.py

```diff
@@ -298,6 +298,7 @@ def _args_from_interpreter_flags():
         'verbose': 'v',
         'bytes_warning': 'b',
         'quiet': 'q',
+        #'py2x_warning': '2',
```

Comment:
Intentionally commented out?

Reply:
Thanks, it has no effect but I may have to just remove it rather than comment.

Reply:
Unless there's a good reason for commented code, I prefer to delete it (commented code is a sort-of phobia of mine!).